tests/unit/assessment/test_jobs.py

```diff
@@ -684,3 +695,77 @@ def test_job_crawler_with_no_owner_should_have_empty_creator_name():
     JobsCrawler(ws, mockbackend, "ucx").snapshot()
     result = mockbackend.rows_written_for("hive_metastore.ucx.jobs", "append")
     assert result == [JobInfo(job_id="536591785949415", job_name="Unknown", creator=None, success=1, failures="[]")]
+
+
+@pytest.mark.parametrize(
+    "start_dttm, days_history",
+    [
+        (datetime(2021, 1, 1, 0, 0, 0), 1),
+        (datetime(2017, 1, 1, 0, 0, 0), 7),
+        (datetime(2025, 1, 1, 0, 0, 0), 30),
+    ],
+    ids=["1_day", "7_days", "30_days"],
+)
+def test_submit_runs_crawler(start_dttm: datetime, days_history: int):
+    ws = create_autospec(WorkspaceClient)
+    days_history = 1
+
+    with patch.object(SubmitRunsCrawler, "_get_current_dttm", return_value=start_dttm):
```

Comment:
this is broken. don't monkey patch databricks code. especially private members. use proper mocks.

Reply:
I'm not patching databricks code. SubmitRunsCrawler is my code, not Databricks code.